### PR TITLE
Invalidate mutation

### PR DESCRIPTION
**Pull request by byronholmes2018**
# Description

Describe change or new feature here.

## Checklist

Remove any items which are not applicable.

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works AND the tests pass

### CODE REVIEW

**Comment by byronholmes2018** [comment]
We can merge it after we get the credHash added to the CredentialType.

File: schema/mutation.js [hunk 1]
```diff
@@ -204,7 +208,161 @@ const Mutation = new GraphQLObjectType({
...
+            }
+        }
+      }, //invalidateCredential
+      validateCredential: { 
+        type: CredentialType, 
+        description: "Validates an invalidated Credential", 
+        args: {
+          id: {
+            type:new GraphQLNonNull(GraphQLID),
+            description: 'The unique id of the credential to be invalidated'
```
Comment (joepound) [inline schema/mutation.js]: Line 302 - perhaps you meant "to be validated"?

File: schema/mutation.js [hunk 2]
```diff
@@ -204,7 +208,161 @@ const Mutation = new GraphQLObjectType({
...
+            type: GraphQLID,
+            description:
+              'USER id associated with the school issuing the new credential'
+            // ^^^ This is the id in the 'users' table
+          }
+        }, 
+        async resolve(parent, args) {
+          if (!args.id || typeof Number(args.id) !== 'number') {
+            return new Error('Please include a credential ID and try again.');
+          }
```
Comment (joepound) [inline schema/mutation.js]: Line 353 - if I recall correctly, ID's in GraphQL can end up as strings (despite looking like numbers), so maybe we instead throw in a check for if the credential could not be found?

Comment (nwthomas) [reply schema/mutation.js]: Yes, we need to change that. I think just the lefthand side of that `or` binary operator would be sufficient.

File: schema/mutation.js (hunk 1 above)
Comment (byronholmes2018) [reply schema/mutation.js]: Yes, thank you! Fixed.

File: schema/mutation.js [hunk 3]
```diff
@@ -204,7 +212,177 @@ const Mutation = new GraphQLObjectType({
           .catch(err => ({ error: err }));
       }
     }, // Remove Credential
- 
+    invalidateCredential:{
```
Comment (nwthomas) [inline schema/mutation.js]: Excellent addition - GraphQL syntax looks good

**Pull request merged**